.github/workflows/deploy-pipeline.yml

```
@@ -0,0 +1,411 @@
+        ssh -o StrictHostKeyChecking=no ${{ secrets.STAGING_USER }}@${{ secrets.STAGING_HOST }} << 'EOF'
+          # Run smoke tests on staging
+          curl -f http://staging.church.yourdomain.com/api/health
+          curl -f http://staging.church.yourdomain.com/
+          echo "Staging tests passed!"
+        EOF
+
+    - name: Notify Staging Deployment
+      uses: 8398a7/action-slack@v3
+      with:
+        status: ${{ job.status }}
+        webhook_url: ${{ secrets.SLACK_WEBHOOK }}
```

actionlint (1.7.7)

input "webhook_url" is not defined in action "8398a7/action-slack@v3". available inputs are "author_name", "channel", "custom_payload", "fields", "github_base_url", "github_token", "icon_emoji", "icon_url", "if_mention", "job_name", "mention", "status", "text", "username"

(action)